src/app/pages/news/9.3/02-config-files.md

```diff
@@ -1,19 +1,8 @@
 ## Config files {#project93}
 
-... SCM ready, easy DevOps, new tree, ...
-
-<div class="short-links">
```

Comment:
linking is indeed a good question: maybe we should start with a file-reference as we do for engine configs e.g.
https://dev.axonivy.com/doc/nightly/engine-guide/configuration/files/app-yaml.html
would fit nicely into 
https://dev.axonivy.com/doc/nightly/designer-guide/configuration/index.html
... but just an idea: we can do it now or later